server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -697,7 +704,7 @@
   "Count_Person_Female_WhiteAlone": "Female Population Identifying as White Alone",
   "Count_Person_Female_WhiteAloneNotHispanicOrLatino": "White Non Hispanic Female Population",
   "Count_Person_Female_WhiteAloneOrInCombinationWithOneOrMoreOtherRaces": "Female White Population, Monoracial/Multiracial",
-  "Count_Person_Female_Widowed": "Widowed Women",
+  "Count_Person_Female_Widowed": "Widows",
```

Comment:
Do you intend to revert this?  I thought Windowed Women seemed better and changed last night.

Reply:
Reverted